snmp/datadog_checks/snmp/utils.py

```diff
@@ -101,6 +101,12 @@ def oid_pattern_specificity(pattern):
     )
 
 
+def decode_asn1_object(obj):
+    # type: (Any) -> float
```

Comment:
That sounds wrong, it doesn't return float, probably strings?

Reply:
Good catch. Actually `value` here is _also_ an ASN1 instance, so the best we could do is `Any`… So I assume this isn't a useful thing to extract — I'll move it back to a straight import from `pyasn1` in `snmp.py`.